patching/config/package.json

```diff
@@ -31,10 +31,15 @@
     "lint": "eslint \"src/**/*.ts\" \"test/**/*.ts\""
   },
   "dependencies": {
-    "@pnpm/patching.types": "workspace:*"
+    "@pnpm/dependency-path": "workspace:*",
+    "@pnpm/error": "workspace:*",
+    "@pnpm/logger": "workspace:*",
```

Comment:
this should be a peer dep.

Reply:
https://github.com/pnpm/pnpm/pull/9236/commits/4e4011d6c13e17b0558a825458d4b1908c94823c